### ISSUE TITLE
Feature Request: Option to hide from categories if bundled into a label

### ISSUE THREAD

**Issue by bgiesing** [leggett/simplify#650]
Any chance you can add an option to remove emails from the category bundles if they also get label bundled?

It completely defeats the purpose of making labels to bundle them up if they still are going to show up under Updates, Promotions, etc. cluttering up the inbox with duplicates. Label bundles should take priority over category bundles instead of appearing in both.

**Comment by leggett** [comment]
Thanks for the request. A few thoughts:

I've sort of run into this myself and have the Updates and Forums categories hidden from the list (so it doesn't bundle). But I don't have that with Social or Promotions. Do you have overlap with your labels/filters and Social or Promotions? 

I think not bundling emails into a category because it has another label could be confusing or worse, lead to unintended consequences. It also is impossible to support with the current setup (linking to a search) which I aim to continue supporting. It would be possible with inline expanding bundles (coming soon) but I don't want bundles to work different depending on your settings. So the question is how to best address your situation.

Separately (but related), I've also figured out how to get Travel, Purchases, and Finance to show up so I can bundle those buckets. This makes the Updates bundle even less useful as it was such a broad category that grabbed a high percentage of my email.

**Comment by bgiesing** [comment]
I have all of the categories enabled and I use all of them, every single one has overlap for me. 

Basically the labels are for specific emails that I get a lot of the same type (so like a label for a specific newsletter) while everything else that I only get occasional emails for (like the email I got that you replied to this issue) gets put into the categories automatically by Gmail cause it's not worth creating a whole new label and filter just to sort out emails I might get at most 1 a day or week vs ones I get 3-10.

It's exactly how I used Inbox when it was still here, nothing in a label ever showed up in the other categories. It made it way easier to get through newer email even if you had a bit of a backlog of old stuff. I could go through the categories and those often would have the more important, smaller emails while the labels had longer emails I could put off for a few days like Newsletters.

With the current setup instead, I'm basically forced to touch the stuff like newsletters first cause I can't find the other email "hiding" in between the others until I clear them out first, and by the time I do clear out those newsletters I'm literally having eye strain issues from reading for over 2 hours straight trying to quickly get through them so I can actually see what other email I've gotten only to not actually be able to read those cause the eye strain and by the time I can get back to it, so many more emails have come in I have to clear those and the cycle keeps repeating, I never end up seeing any of the important messages cause I'm spending so long clearing out optional stuff cause it's burying everything else.

One possible solution would be to use the minus filter, could autodetect any other labels in a category so when you click on it instead of searching for `category:updates`, you search for `category:updates -label:newsletters`

**Comment by leggett** [comment]
I'll think on it some. 

Detecting the labels on load of the search and then redirecting to a new search minus the labels adds lag on initial load and if you go to the next page of results (where I would have to redirect again). 

Since you have filters to apply the label, can you not also set them to put them in the Primary category? This isn't an automatic fix and maybe you have a lot of filters. But I think it gives you the experience you want. 

I believe this is how it worked in Inbox (any filter that filtered it into a label, would automatically pull it from any category). But I'm honestly not 100% sure.

**Comment by bgiesing** [comment]
> Since you have filters to apply the label, can you not also set them to put them in the Primary category? This isn't an automatic fix and maybe you have a lot of filters. But I think it gives you the experience you want.

Yeah, this somewhat works with one downside (if there's only one single email in the label, it now shows up inside the main inbox instead of falling back into the categories) but that doesn't happen that often so it's a good workaround. A bit annoying since I have like 50 filters, but I can edit them over time as I get email in.

**Comment by leggett** [comment]
I'm adding the option to bundle threads even if there is only one in that category or label in the next update. (Right now, there has to be at least two before I bundle.)
